tools/bin/commands/util/openshift_funcs

```diff
@@ -13,8 +13,10 @@ check_oc_version()
     # Removes any lines containing kubernetes or server
     # Extracts any version number of the format dd.dd.dd, eg. 3.10.0 or 4.1.0
     #
-    local test=$(oc version | grep -Eiv 'kubernetes|server' | grep -o '[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}\?')
+    set_gnugrep
 
+    local test=$(oc version | $gnugrep -Eiv 'kubernetes|server' | $gnugrep -o '[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}\?')
```

Comment:
I'm thinking rather than introducing the dependency on gnu grep, and make OSX folk install it if they don't have we could perhaps eliminate the need upon it. Would that be better? We could probably parse this out differently. For example for OpenShift 4.2.x there's a much nicer `oc version` implementation (https://github.com/openshift/oc/pull/5).

Reply:
I think `oc version|awk '/^oc v/ {sub(/^oc v/, ""); sub(/^oc v/, ""); sub(/\+.*/, ""); print}/Server Version/{print $3}'` should work. I'll create a PR...